tbx/project_styleguide/templates/patterns/pages/impact_reports/impact_report_page.html

```diff
@@ -0,0 +1,27 @@
+{% extends "patterns/base_page.html" %}
+{% load wagtailcore_tags wagtailimages_tags static %}
+
+{% block content %}
+
+    {% include "patterns/molecules/hero/hero.html" with title=page.strapline desc=page.intro in_page_nav=True %}
+
+    <ul>
+        {% for heading in page.headings %}
+            <li><a href="#{{ heading.slug }}">{{ heading.short_heading }}</a></li>
+        {% endfor %}
+    </ul>
+
+
+    <div id="{{ page.introduction_title|slugify }}" style="padding-top: 50px;">
+        <h2>{{ page.introduction_title }}</h2>
+    </div>
+
+    {% for page_author in page.authors.all %}
+        {% include "patterns/molecules/author/author.html" with author=page_author.author item=page tag_link_base=page.blog_index.url %}
```

Comment:
use `pageurl` where possible
```suggestion
        {% pageurl page.blog_index as tag_link_base %}
        {% include "patterns/molecules/author/author.html" with author=page_author.author item=page tag_link_base=tag_link_base %}
```

Having said that, `ImpactReportPage` doesn't have a blog_index attribute. I take it you modelled this on the blog page which has
https://github.com/torchbox/wagtail-torchbox/blob/master/tbx/project_styleguide/templates/patterns/pages/blog/blog_detail.html#L34 and 
https://github.com/torchbox/wagtail-torchbox/blob/master/tbx/blog/models.py#L238